src/components/ResultsHeaderContainer/ResultsHeaderContainer.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+import { QuizResultsConfig } from '../../types';
+
+export interface IResultHeaderContainerProps {
+  resultsConfig?: QuizResultsConfig | null;
+  hasNoResults: boolean;
+}
+
+export default function ResultsHeaderContainer({
+  hasNoResults,
+  resultsConfig,
+}: IResultHeaderContainerProps) {
+  let resultsTitle: string;
```

Comment:
Let me know if this doesn't happen for you but the results config is not being fetched for me at the moment

![image](https://github.com/Constructor-io/constructorio-ui-quizzes/assets/2954126/2e60b472-2589-4aab-ac4f-98c447f946bc)

Reply:
I just relaunched the app and the config is being fetched for me. Can you share the flow where you got the error?

<img width="1821" alt="Screenshot 2023-12-26 at 9 52 33 PM" src="https://github.com/Constructor-io/constructorio-ui-quizzes/assets/68158388/691c98ad-4eae-4873-a4d7-677f661b3a1d">

Reply:
Also on refresh
<img width="1821" alt="Screenshot 2023-12-26 at 10 51 27 PM" src="https://github.com/Constructor-io/constructorio-ui-quizzes/assets/68158388/15982a2b-471d-4bf4-82b9-0ca0f6e6a240">